Resources/Help/analyses/ldexponential_nl.md

```diff
@@ -1,66 +1,60 @@
+- Empirische cumulatieve verdeling: Geeft een empirische cumulatieve verdelingsgrafiek van de geselecteerde variabele weer.
 
-## Parameters schatten
-- Maximale waarschijnlijkheid: pas de theoretische verdeling aan de geselecteerde variabele aan door de waarschijnlijkheidsfunctie te maximaliseren. De parameterwaarden die zijn gespecificeerd in het gedeelte Verdeling tonen dienen als startpunt voor de procedure.
+## Schat Parameters
+- Meest aannemelijke schatter: Schat de parameters met de waarden in het domein waar de aannemelijksheidsfunctie maximaal is. De aannemelijksheidsfunctie zet het gegevensargument (op basis van de geselecteerde variabele) in de theoretische kansdichtsheidsfunctie vast, en beschouwt deze als een functie van de parameters. De optimalisatieprocedure wordt geïnitialiseerd met de waarden voor de parameters die zijn ingevoerd onder "Toon Verdeling".
 
 ### Uitvoer
-- Schattingen: geeft een tabel weer met de parameterschattingen. Het wijzigen van parametrisering verandert welke parameters worden weergegeven.
-- Soa. error: Geeft de standaardfout van de parameterschattingen weer
-- Betrouwbaarheidsinterval: geeft het betrouwbaarheidsinterval van de parameters weer op het opgegeven betrouwbaarheidsniveau.
-
+- Schattingen: Geeft een tabel weer met de parameterschattingen. De getoonde parameters passen zich aan aan de gekozen parametrisatie.
+- Std. error: Geeft de standaardfout van de parameterschattingen weer.
+- Betrouwbaarheidsinterval: Geeft het betrouwbaarheidsinterval van de parameters weer op het opgegeven betrouwbaarheidsniveau.
 
-## Fit beoordelen
+## Evalueer Passing
 
-### Plots
-- Histogram vs. theoretische pdf: geeft een histogram weer van de geselecteerde variabele overlay met de kansdichtheidsfunctie van de gepaste distributie
-- Empirische vs. theoretische cdf: Geeft een empirische cumulatieve verdelingsgrafiek weer, overlay met de cumulatieve verdelingsfunctie van de gepaste distributie
-- Q-Q-plot: geeft de kwantiel-kwantielgrafiek weer. De * x * -as toont de theoretische kwantielen van de gegevenspunten onder de gepaste verdeling, de * y * -as toont de empirische kwantielen van de geselecteerde variabele.
-- P-P-plot: geeft de kans-waarschijnlijkheidsgrafiek weer. De * x * -as toont de theoretische waarde van de cumulatieve dichtheidsfunctie van de gegevenspunten onder de gepaste verdeling, de * y * -as toont de empirische percentielen van de geselecteerde variabele.
+### Grafieken
+- Histogram vs. theoretische pdf: Geeft een histogram weer van de geselecteerde variabele en ook de kansdichtheidsfunctie van de gepaste distributie.
+- Empirische vs. theoretische cdf: Geeft een empirische cumulatieve verdelingsgrafiek weer, en ook de cumulatieve verdelingsfunctie van de gepaste distributie.
+- Q-Q-plot: Geeft de kwantiel-kwantielgrafiek weer. De *x*-as toont de theoretische kwantielen van de gepaste verdeling, de *y*-as toont de empirische kwantielen van de geselecteerde variabele.
```

Comment:
Lijkt me dat dit nu juist  'kwantiel-kwantiel grafiek' moet zijn of als in het engels met 2 streepjes.
kwantiel-kwantiel (los door de Q-Q)

Reply:
Het is nu Q-Q plot: kwantiel-kwantielgrafiek

Reply:
Moet het eigenlijk niet 

Q-Q grafiek

of 

Q-Q diagram:

zijn?